doc/builders/builders.chapter.md

```diff
@@ -0,0 +1,56 @@
+# The builders {#chap-builders}
+
+A builder is a function that produces derivations.
```

Comment:
Looks like the suggestions and discussion about wording and naming got lost track of. How about we split out the new pages into a separate PR so we don't block the functional changes on docs?

Reply:
Does that mean splitting the commits related to the Nixpkgs manual into another PR?

I'll try that after pushing the latest progress.

Reply:
Now the documentation changes are split into #262648.